STT/run.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/sh
+
+set -xe
+
+THIS=$(dirname "$0")
+export PATH=${THIS}:${THIS}/${MODEL_LANGUAGE}:$PATH
+
+export TF_CUDNN_RESET_RND_GEN_STATE=1
```

Comment:
Do we still need this ?

Reply:
Yes it doesn't train without it otherwise.

Reply:
I think it's only needed for 30+ series GPUs though. We could use something like that to figure out if we should enable random gen states:

```bash
lspci | grep VGA | cut - -d ":" -f3 | sed -e 's/^.*\[//;s/\].*$//' | grep -o -E "[0-9]+" | cat | tr '\n' ' ' | <<< '/ 100' | bc | cat |  tr '\n' ' ' | <<< '- 29' | bc
```

First `lspci | grep VGA | cut - -d ":" -f3` grep the card name and model:
```zsh
❯ lspci | grep VGA | cut - -d ":" -f3
 NVIDIA Corporation GA106 [GeForce RTX 3060] (rev a1)
```

Then with `sed -e 's/^.*\[//;s/\].*$//' | grep -o -E "[0-9]+" | cat` we just ask for the series of the cards:
 ```zsh
❯ lspci | grep VGA | cut - -d ":" -f3 | sed -e 's/^.*\[//;s/\].*$//' | grep -o -E "[0-9]+" | cat
───────┬────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
       │ STDIN
───────┼────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
   1   │ 3060
───────┴────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
```

And finally we make sure to return 0 or less if it's not concerned or 1 or more if it applies using `tr '\n' ' ' | <<< '/ 100' | bc | cat |  tr '\n' ' ' | <<< '- 29' | bc`:
```zsh
❯ lspci | grep VGA | cut - -d ":" -f3 | sed -e 's/^.*\[//;s/\].*$//' | grep -o -E "[0-9]+" | cat | tr '\n' ' ' | <<< '/ 100' | bc | cat |  tr '\n' ' ' | <<< '- 29' | bc
1
```

This means I have a 3060  which is affected. Idem if I had a 40 something.

Reply:
It looks like this in the dockerfile now:
```dockerfile
# Configure random state
# Required for trainig on newer GPUs such as series 30/40.
# You safely disable it (set to 0) if your GPU doesn't need it.
ARG use_tf_random_gen_state=1
ENV TF_CUDNN_RESET_RND_GEN_STATE=$use_tf_random_gen_state

```